serve-api/internal/db/db.go

```diff
@@ -0,0 +1,31 @@
+package db
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/ministryofjustice/serve-opg/serve-api/entity"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+func Connect() *gorm.DB {
+	db_host := os.Getenv("POSTGRES_HOST")
+	db_user := os.Getenv("POSTGRES_USER")
+	db_pswd := os.Getenv("POSTGRES_PASSWORD")
+	db_name := os.Getenv("POSTGRES_DB")
+
+	dsn := fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=5432 sslmode=disable", db_host, db_user, db_pswd, db_name)
```

Comment:
You may need to parameterise sslmode as well for the move to production.

Reply:
We'll pick this up as part of the ECS ticket seeing as the code we have right now isn't being built.